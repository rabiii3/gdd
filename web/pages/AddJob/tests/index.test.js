import React from 'react';
import { mount } from 'enzyme';
import { Provider } from 'react-redux';
import configureStore from '../../../store/configureStore.test';
import { enhance } from '..';

describe( 'app | pages | AddJobDescription | index', () => {
  it('should match snapshot', () => {
    const Component = () => <div/>;
    const EnhancedComponent = enhance(Component);
    const state = {};
    const store = configureStore(state);
    const wrapper = mount(
      <Provider store={store}>
        <EnhancedComponent />
      </Provider>
    );
    expect(wrapper).toMatchSnapshot();
  });
});