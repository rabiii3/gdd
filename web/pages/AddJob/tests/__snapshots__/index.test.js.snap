// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app | pages | AddJobDescription | index should match snapshot 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <WithStyles(Component)>
    <Component
      classes={
        Object {
          "root": "Component-root-1",
        }
      }
    >
      <div />
    </Component>
  </WithStyles(Component)>
</Provider>
`;
